pkg/testutil/fakes.go

```diff
@@ -337,3 +337,62 @@ func (s simpleServiceEntryNamespaceLister) List(selector labels.Selector) (ret [
 func (s simpleServiceEntryNamespaceLister) Get(name string) (*v1alpha3.ServiceEntry, error) {
 	return s.istioClientSet.NetworkingV1alpha3().ServiceEntries(s.namespace).Get(context.TODO(), name, metav1.GetOptions{})
 }
+
```

Comment:
We have been adding tests for the listers in fakes_test.go.  Not sure if we need to be adding tests for the test code though.